riff-raff/app/conf/context.scala

```diff
@@ -269,98 +255,3 @@ class Config(configuration: TypesafeConfig, startTime: DateTime) extends Logging
 
   override def toString: String = configuration.toString
 }
-
-class Management(config: Config, shutdownWhenInactive: ShutdownWhenInactive, deployments: Deployments, datastore: DataStore) {
```

Comment:
We collected a load of metrics using Guardian Management that look interesting but aren't actually in use from what I can tell. Open Telemetry/something else is probably the future here.